### ISSUE TITLE
normalized entropy different in paper vs code?

### ISSUE THREAD

**Issue by RutgerEttes** [byungdoh/attn_dist#1]
Hi, while I was looking at the the code for calculating the normalized attention entropy as well as the paper to understand how it worked, I noticed that in line 52 of `main.py`: 
`    return torch.sum(torch.div((-1 * tensor * log_tensor), norm_tensor), dim=-1)`
there seems to be an extra multiplication with -1 that I can't find in the paper.

Am I missing something? I think the -1 does bring it closer to the definition of entropy, but it also seems to be missing from the definition of attention entropy of Ryu and Lewis.

**Comment by byungdoh** [comment]
Thanks for pointing this out! I think what happened is that I carried over Ryu and Lewis's (2021) mistake in the equation directly to my paper.

You're right that the -1 is necessary to calculate entropy, which the codebase correctly reflects.
